README.md

```diff
@@ -56,7 +56,28 @@ It has specific TDR commands around realm import, and is copied from the default
 
 The standalone-ha.xml is mostly the standard configuration for keycloak with a few changes to get it to work with the load balancer. Some of these are discussed in the keycloak [documentation](https://www.keycloak.org/docs/latest/server_installation/#_setting-up-a-load-balancer-or-proxy)
 
-## Updating Keycloak Configuration json
+## Updating TDR Realm Configuration json
+
+A separate Jenkins job is used to update the TDR realm configuration: TDR Auth Server Update
+
+This is because on container start up, if a realm already exists, its configuration json is ignored. So restarting the container will not update the configuration.
+
+The TDR Auth Server Update Jenkins job updates existing realm configuration using the Keycloak REST APIs: https://www.keycloak.org/docs-api/11.0/rest-api/index.html
+
+Two REST endpoints are used to update a Keycloak realm:
+* `PUT /{realm}`: updates the top-level information of the realm.    
+  * Takes the full realm representation: https://www.keycloak.org/docs-api/11.0/rest-api/index.html#_realmrepresentation
+  * Any user, roles or client information in the representation are ignored.
+* `POST /{realm}/partialImport`: Partial import from a JSON file to an existing realm.
+  * Takes a partial realm representation: https://www.keycloak.org/docs-api/11.0/rest-api/index.html#_partialimportrepresentation
+  * Allows importation of configuration for: `clients`, `roles`, `groups` and `identity providers` only
+  * Uses a policy option that controls what behaviour occurs for existing realm resources:
+    * `SKIP`: Skips any existing resources and does not update them
+    * `FAIL`: Causes the whole import to fail
```

Comment:
When would we want to use this option in Jenkins?

Reply:
Do you mean the "FAIL" option? If yes, then I can't think of a explicit use case. Though I was using this option when testing the Jenkins job worked, without making any updates to Keycloak itself. Can remove it easily enough if not needed.